### DISCUSSION TITLE
Feature request: PS1 as source for intune

### DISCUSSION THREAD

**Discussion by X-15** [rink-turksma/IntunePrepTool#4] [Ideas]
Pretty much the title, would save a lot of time.

**Comment by rink-turksma** [comment]
Hi, I will fix this somewhere this week.

**Comment by rink-turksma** [comment]
Hi X-15,

Please see version: https://github.com/rink-turksma/IntunePrepTool/releases/tag/IntunePrepTool-1.4.8.0
Is this what your looking for?

Greetings,
Rink
